src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { MoviesListShape } from './MoviesListShape';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <MovieCard movie={movie} key={movie.imdbId} />
```

Comment:
І тоді ти зможеш перевикористати MovieCardShape в MovieListShape
і не повторюватися
```suggestion
      <MovieCard { ...movie } key={movie.imdbId} />
```